Add Cody Commands docs in VS Code (#56112)
sourcegraph/sourcegraph: doc/cody/capabilities.md

- Select **Cody by Sourcegraph** from the dropdown list

![View Cody's autocomplete logs from the Output View in VS Code](https://storage.googleapis.com/sourcegraph-assets/Docs/view-autocomplete-logs.png)

## Commands

> Cody supports **Commands** with its [VS Code extension](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai). These allow you to create and run reusable prompts without submitting pull requests or opening the chat sidebar.

Reviewer: ```suggestion
Cody supports executing reusable prompts known as **Commands** from within the [VS Code extension](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai). They allow you to run predefined actions (like `Explain Selected Code` and `Generate Unit Tests` for example) with smart context-fetching anywhere in the editor.
```

> You can run a command in 3 ways:

Reviewer: ```suggestion
There are three ways to run a Cody Command:
```

> 1. Type `/` in the chat bar. Cody will then suggest a list of commands

Reviewer: ```suggestion
1. Type `/` in the chat bar. Cody will then suggest a list of available commands
```

> 2. Right click > Cody > Select a command

Reviewer: ```suggestion
2. Right click and select `"Cody"` > Choose a command from the list.
```

> 3. Press the command hotkey (`⌥` + `C` / `Alt` + `C`)

Reviewer: ```suggestion
3. Use the predefined command hotkey: `⌥` + `C` / `Alt` + `C`
```

![Cody Commands in VS Code](https://storage.googleapis.com/sourcegraph-assets/Docs/cody-commands.png)

> ### Custom Commands

Reviewer: ```suggestion
### Custom Commands <span class="badge badge-experimental">Experimental</span>
```

> The **Custom Commands** feature lets you customize Cody's abilities for your projects without writing code. This provides a flexible way to create reusable "commands" that give Cody targeted instructions and context from your project to enhance productivity.

Reviewer: ```suggestion
The **Custom Commands** feature allows you to customize Cody's abilities for your specific projects and workflow without writing code. They are defined in JSON format and let you call CLI tools, write custom prompts, and configure context sent to Cody. This provides a flexible way to create reusable "commands" tailored to your needs.
```

> You can define customized prompts using `JSON` files that can be invoked simply by typing a `/` in the chat box or using keyboard shortcuts `Shift+Cmd+V` without opening the chat sidebar.

Reviewer: ```suggestion
You can define customized prompts using `JSON` files that can be invoked simply by typing a `/` in the chat box or using keyboard shortcuts `⌥` + `C` / `Alt` + `C` without opening the chat sidebar.
```

Right-click on the selected code to open the Cody context menu and select the **Custom Commands (Experimental)** option.

![Cody Custom Commands in VS Code](https://storage.googleapis.com/sourcegraph-assets/Docs/create-custom-commands.png)

### `cody.json` file

> You can define custom commands for Cody in the `cody.json` file. To make commands accessible only in a specific project, create the `cody.json` file in the `.vscode` directory of your project. Cody will execute these workspace-specific custom commands when working in that project.

Reviewer: ```suggestion
You can define custom commands for Cody in a `cody.json` file. To make commands only available for a specific project, create the `cody.json` file in the `.vscode` directory of that project. These workspace-specific custom commands will be available to you when you work on that project.
```

> To make commands accessible across multiple projects, create a new `cody.json` file in the `.vscode` directory of your home directory. Cody will execute these global custom commands in any workspace.

Reviewer: ```suggestion
To make custom commands globally available across multiple projects, create a new `cody.json` file in the `.vscode` folder of your home directory. These global custom commands will be available for you in Cody in any workspace.
```
